docs/index.rst

```diff
@@ -9,6 +9,7 @@ Manticore is a prototyping tool for dynamic binary analysis, with support for sy
 
    api
    models
+   internal
```

Comment:
I mostly agree with you. I ended up adding them here when you asked about a docstring rendering correctly in Sphinx. Vote to remove?